src/notebookKernel.ts

```diff
@@ -56,6 +105,19 @@ export class NotebookKernel implements vscode.NotebookKernel {
     const kernelOutputs = outputStream
       .pipe(
         reduce((acc: vscode.CellOutput[], msg): vscode.CellOutput[] => {
+          if (isMessageType('execute_result', msg)) {
+            return [
+              ...acc,
+              {
+                outputKind: vscode.CellOutputKind.Rich,
+                data: (msg.content as {
+                  data: {
+                    [mimeType: string]: string;
+                  }
+                }).data
```

Comment:
You can remove the need for the cast by adding the execute_result type to this union https://github.com/microsoft/vscode-simple-jupyter-notebook/blob/a7f9b00e0025c33a3086d50d0b069c33c2f1a796/src/messaging.ts#L153